README.md

```diff
@@ -12,11 +12,9 @@ Docker image that runs the Dash dashd node in a container for easy deployment.
 Requirements
 ------------
 
-* Physical machine, cloud instance, or VPS that supports Docker (i.e. Vultr, Digital Ocean, KVM or XEN based VMs) running Ubuntu 14.04 or later (*not OpenVZ containers!*)
-* At least 4 GB to store the block chain files
-* At least 1 GB RAM + 2 GB swap file
-
-Recommended and tested on Vultr 1024 MB RAM/320 GB disk instance @ $8/mo.  Vultr also *accepts Bitcoin payments*!  May run on the 512 MB instance, but took *forever* (1+ week) to initialize due to swap and disk thrashing.
+* Physical machine, cloud instance, or VPS that supports Docker (i.e. Vultr, Digital Ocean, KVM or XEN based VMs) running Ubuntu 16.04 or later (*not OpenVZ containers!*)
+* At least 20 GB to store the block chain files
```

Comment:
Per https://docs.dash.org/en/stable/masternodes/understanding.html#masternode-requirements, minimum disk is now 40 GB.


  | Minimum | Recommended
-- | -- | --
CPU | 1x 1 GHz | 1x 2 GHz
RAM | 2 GB + 2 GB swap | 4 GB + 2 GB swap
Disk | 40 GB | 60 GB
Network | 400 GB/mth | 1 TB/mth